### DISCUSSION TITLE
The developer of play-services-safetynet (com.google.android.gms:play-services-safetynet) added the following note to SDK version 18.0.0:

### DISCUSSION THREAD

**Discussion by Abjithwyn** [firebase/flutterfire#11824] [Q&A]
The developer of play-services-safetynet (com.google.android.gms:play-services-safetynet) added the following note to SDK version 18.0.0:
The SafetyNet Attestation API is being discontinued and replaced by the new Play Integrity API. Begin migration as soon as possible to avoid user disruption. The Play Integrity API includes all the integrity signals that SafetyNet Attestation offers and more, like Google Play licensing and better error messaging. Learn more and start migrating at https://developer.android.com/training/safetynet/deprecation-timeline

I have been seeing this warning message for a while when I upload my appbundle to playstore, I did some digging and came to know that firebase auth is causing this. anyone else facing this issue...??

**Comment by FullstackWEB-developer** [comment]
I got same issue
It was helpful:
https://stackoverflow.com/questions/73245130/how-to-fix-remove-the-play-services-safetynet-warning

**Reply by Abjithwyn** [reply]
thanks

**Comment by kitterive** [comment]
when I  added this in android/app/build.gradle
   configurations.all { 
    exclude group: 'com.google.android.gms', module: 'play-services-safetynet'
}
I got error :rror:  r8: missing class com.google.android.gms.safetynet.safetynet   from firebase sdk

**Comment by huzaifak08** [comment]
Add this to app level build.gradle:

`configurations.all {
    exclude(group = "com.google.android.gms", module = "play-services-safetynet")
}`
